buildout.cfg

```diff
@@ -1,13 +1,14 @@
 [buildout]
 develop = .
-parts = interpreter test
+parts = interpreter test 
 
 [interpreter]
 recipe = zc.recipe.egg
 eggs = zope.globalrequest
-interpreter = python
+interpreter = zopepy
```

Comment:
If ``py`` is preferred we can change it. In Plone world I saw a lot ``zopepy``, so I dont care as long as it does not collide.